utils/dashboard_creators/static/styles.css

```diff
@@ -3,9 +3,12 @@
     position: relative;
     display: inline-block;
   }
+td:last-child,td:nth-last-child(2),td:nth-last-child(3),td:nth-last-child(4),td:first-child{
```

Comment:
Perhaps you can add a comment above on what this serves?

Reply:
blocks color coding non metrics. moved it to a better place now.